src/openxr/OpenXRApi.cpp

```diff
@@ -852,9 +893,19 @@ bool OpenXRApi::initialiseSession() {
 		return false;
 	}
 
+#ifdef ANDROID
+	XrSystemColorSpacePropertiesFB color_space_properties_fb = {
```

Comment:
I wouldn't switch this on Android but based on whether the color space extension is available. Other vendors may end up implementing this extension as well

Reply:
Let me add a todo for this one as well. It's switched on Android for now because it's one of the required extension in the Android switch above.
Once we switch to the new mechanism, then it'll be setup accordingly.

Reply:
Sounds good to me